lib/ts/recipe/session/types.ts

```diff
@@ -129,6 +129,7 @@ export type TypeInput = {
 
 export type TypeNormalisedInput = {
     refreshTokenPath: NormalisedURLPath;
+    accessTokenPath?: NormalisedURLPath;
```

Comment:
the normalised input should not be optional. You should add `accessTokenPath` to the `TypeInput` type, and then in the `validateAndNormaliseUserInput` function [here](https://github.com/supertokens/supertokens-node/blob/1119aa4866/lib/ts/recipe/session/utils.ts#L127), you should set the normalised input to this value if provided by the user, else to `normaliseURLPathOrThrowError("/")`

Reply:
Done